cmd/optimizely/main.go

```diff
@@ -99,6 +99,8 @@ func main() {
 	conf := loadConfig(v)
 	initLogging(conf.Log)
 
+	conf.LogConfigurationWarnings()
```

Comment:
nit: we use the abbv `Config` elsewhere, we should probably do it here as well: `LogConfigWarnings`